core/src/main/java/org/apache/iceberg/ClientPoolImpl.java

```diff
@@ -140,4 +148,12 @@ private void release(C client) {
   public int poolSize() {
     return poolSize;
   }
+
+  public static AtomicInteger openCount() {
+    return openCount;
+  }
+
+  public static AtomicInteger closeCount() {
+    return closeCount;
```

Comment:
Would it make more sense to call `.get()` in the return statement and just return an `Integer`?

It seems the only places in the code that these functions are called, `.get()` is called immediately. It seems to me that calling `.get()` would provide the benefit of a defensive copy to truly leave these private.

Reply:
Thanks，your suggestion is very helpful, i'll do it.